charts/netbox/templates/configmap.yaml

```diff
@@ -29,6 +29,9 @@ data:
       {{- end }}
       OPTIONS:
         sslmode: {{ .Values.externalDatabase.sslMode | quote }}
+        {{- if .Values.externalDatabase.sslRootCert }}
+        sslrootcert: {{ .Values.externalDatabase.sslRootCert | quote }}
+        {{- end}}
```

Comment:
@LeoColomb is there a cleaner way to print the `sslrootcert` option only if the value is defined (like a helper or something) ?